src/Raven.Server/Documents/Indexes/Sharding/ShardedDatabaseIndexCreateController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using JetBrains.Annotations;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Exceptions.Sharding;
+
+namespace Raven.Server.Documents.Indexes.Sharding;
+
+public class ShardedDatabaseIndexCreateController : DatabaseIndexCreateController
+{
+    public ShardedDatabaseIndexCreateController([NotNull] DocumentDatabase database)
+        : base(database)
+    {
+    }
+
```

Comment:
do we need to override the `ValidateStaticIndexAsync` as well?

Reply:
don't thinks it's needed, static indexes are handled by `ShardedIndexCreateController`